openreview/conference/helpers.py

```diff
@@ -61,9 +62,16 @@ def get_conference(client, request_form_id):
         'Submissions should be public, but reviews should be private.' == note.content.get('Open Reviewing Policy', '') :
         builder.set_submission_public(True)
 
+    # 'Custom Submission Fields' doesn't exist in the request form yet,
+    # but it can be added through a script via the
+    # `content_override_args` parameter.
+    additional_fields = note.content.get('Custom Submission Fields', {})
```

Comment:
We should add this field to the form

Reply:
I agree. I think to make this work, we'll need a special widget that lets users easily create fields that are formatted for `content.reply`, since they have a special structure (e.g. some set of options that lets them choose between "values", "values-regex", "values-dropdown", etc.)

Reply:
Yeah, that is still in progress by Zach. In the meantime, I would use value-json, the same widget that we use for Custom Loads in the matching that accepts a json string.